Sources/SwiftParser/RawTokenKindSubset.swift

```diff
@@ -50,37 +48,18 @@ extension RawTokenKindSubset {
   var precedence: TokenPrecedence? {
     return nil
   }
-
-  func accepts(lexeme: Lexer.Lexeme) -> Bool {
-    return true
-  }
-
-  static var allRawTokenKinds: [RawTokenKind] {
-    return self.allCases.map { $0.rawTokenKind }
```

Comment:
This was the main offender for the performance.

Reply:
:oof: